token-syncer/src/token_syncer/commands/syncer.clj

```diff
@@ -236,3 +235,38 @@
     (catch Throwable th
       (log/error th "unable to sync tokens")
       (throw th))))
+
+(def sync-clusters-config
+  {:build-context (fn build-sync-clusters-context
+                    [parent-context {:keys [options]}]
+                    {:context parent-context
+                     :options options})
+   :execute-command (fn execute-sync-clusters-command
+                      [{:keys [context options]} arguments]
+                      (let [{:keys [waiter-api]} context
+                            {:keys [limit]} options
+                            cluster-urls-set (set arguments)]
+                        (cond
+                          (<= (-> cluster-urls-set set count) 1)
+                          {:exit-code 1
+                           :message (str "at least two different cluster urls required, provided: " (vec arguments))}
+
+                          :else
+                          (let [sync-result (sync-tokens waiter-api cluster-urls-set limit)
+                                exit-code (-> (get-in sync-result [:summary :sync :error] 0)
+                                              zero?
+                                              (if 0 1))]
+                            (log/info (-> sync-result pp/pprint with-out-str str/trim))
+                            {:exit-code exit-code
+                             :message (str "exiting with code " exit-code)}))))
+   :option-specs [["-l" "--limit LIMIT" "The maximum number of tokens to attempt to sync, must be between 1 and 10000"
+                   :default 1000
+                   :parse-fn #(Integer/parseInt %)
+                   :validate [#(< 0 % 10001) "Must be between 1 and 10000"]]]
+   :retrieve-documentation (fn retrieve-sync-clusters-documentation
```

Comment:
Parts of this string seem like it will be replicated across subcommands.  Would it be better to extract the portions that vary?

`{:name "sync-clusters"
 :description "syncs tokens among (at least two) Waiter clusters in the URL(s)"}`

Do you expect `retrieve-documentation` to vary otherwise?